### PR TITLE
Prisma Comment Workflow Change

### PR DESCRIPTION
**Pull request by devashish9**
@lukejianu Does this look good to you? Made a minor change so that the comment only shows up in the necessary PRs

### CODE REVIEW

File: .github/workflows/auto-comment.yml [hunk 1]
```diff
@@ -1,5 +1,9 @@
 name: Auto Comment
-on: [issues, pull_request]
+on:
+  pull_request:
+    paths:
```
Comment (lukejianu) [inline .github/workflows/auto-comment.yml:4]: Oh this is cool, I didn't know you could do this. Is there a reason it's set up for any changes in the ``prisma`` directory and not only when the ``schema.prisma`` file changes?

Comment (devashish9) [reply .github/workflows/auto-comment.yml:4]: no particular reason. I can change it to just that file if you prefer

Comment (devashish9) [reply .github/workflows/auto-comment.yml:4]: done

**Review by lukejianu — APPROVED** [review]
LGTM assuming this works.

**Pull request merged**